src/Settings/settings.json

```diff
@@ -657,11 +767,15 @@
         "tip": "The maximum number of characters in a source file to open.\nA source file won't be opened if it's too long."
     },
     {
-        "name": "Load Test Case Length Limit",
+        "name": "Display Test Case Length Limit",
```

Comment:
Although very minor change, I think it should be reflected in the `cpeditor/docs` as well.

Reply:
It won't be in the changelog, but the docs for this setting will be updated.

Reply:
I noticed the name of this setting when reviewing cpeditor/cpeditor.github.io#76

Reply:
> It won't be in the changelog, but the docs for this setting will be updated.

Yes, I mean the doc in the website for English and Chinese (possibly) could be updated to "Display" instead of "Load".

This is just another way to make sure that we keep website doc up to date as we merge PRs.